.github/workflows/causal_lm_cpp.yml

```diff
@@ -597,7 +597,7 @@ jobs:
       - name: Install OpenVINO
         run: |
           mkdir ./ov/
-          curl ${{ env.l_ov_link }} | tar --directory ./ov/ --strip-components 1 -xz
+          curl https://storage.openvinotoolkit.org/repositories/openvino/packages/pre-release/2024.4.0rc2/linux/l_openvino_toolkit_ubuntu24_2024.4.0.dev20240905_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
```

Comment:
Cool idea switching one of the tests to a different OS version. Or is it because chat_sample and lcm_dreamshaper_v7_cpp don't work on Ubuntu20?

Reply:
we need a test with default GHA runner. 
Customer runners are not exposed (probably) on U24.